src/clj/fluree/db/query/api.cljc

```diff
@@ -34,15 +34,19 @@
     did (assoc :did did :issuer did)))
 
 (defn restrict-db
-  [db t opts]
+  [db t {:keys [did default-allow?] :as opts}]
   (go-try
-    (let [db*  (if-let [did (:did opts)]
-                 (<? (perm/wrap-identity-policy db did true nil))
-                 db)
-          db** (-> (if t
-                     (<? (time-travel/as-of db* t))
-                     db*))]
-      (assoc-in db** [:policy :cache] (atom {})))))
+   (let [db*  (if did
+                (<? (perm/wrap-identity-policy db
+                                               did
+                                               (or (true? default-allow?)
```

Comment:
Is this `or` expression even necessary? I think it just boils down to `default-allow?`

Reply:
That's a good point... let me look.

Reply:
Added malli option here 7b3d4d51bc8c9268d584e9b1af7902aee90d4c73 , and updated params in 1eea5cb038b5ac7b38a538e7641a46aa224c5170